www/assessment/customization/style-guide/webpack.config.base.js

```diff
@@ -0,0 +1,137 @@
+/* globals require, __dirname */
+const path = require('path');
+const webpack = require('webpack');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const autoprefixer = require('autoprefixer');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
Remove all this, if you are going to use predefined Libraries for UI like Material UI, keep this set up as small as possible